hack/generate-swagger.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+source "${REPO_ROOT}"/hack/util.sh
+
+util::verify_go_version
+
+go run hack/tools/swagger/generateswagger.go > api/openapi-spec/swagger.json
+
+# Delete trash of generating swagger doc
```

Comment:
`defer os.Remove()` in `render.go` can't delete content of `/tmp`, so remove it here.